lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -56,12 +56,12 @@
       </a>
         % else:
         <a class="fade-cover">
-        <img src="${course_image_url(course)}" alt="${_('{course_number} {course_name} Cover Image').format(course_number=course.number, course_name=course.display_name_with_default) |h}" />
+        <img src="${course_image_url(course)}" alt="${_('{course_number} {course_name} Home Page').format(course_number=course.number, course_name=course.display_name_with_default) |h}" />
```

Comment:
@cptvitamin can you chime in here? Originally we wanted to remove the `alt` text from the image, but it's its own link so should we keep it?